src/Tribe/Attendees_Table.php

```diff
@@ -134,8 +134,8 @@ public function column_cb( $item ) {
 	 * @return string
 	 */
 	public function column_primary_info( array $item ) {
-		$purchaser_name  = empty( $item[ 'purchaser_name' ] ) ? '' : esc_html( $item[ 'purchaser_name' ] );
-		$purchaser_email = empty( $item[ 'purchaser_email' ] ) ? '' : esc_html( $item[ 'purchaser_email' ] );
+		$purchaser_name  = empty( $item['holder_name'] ) ? ( empty( $item[ 'purchaser_name' ] ) ? '' : esc_html( $item[ 'purchaser_name' ] ) ) : $item['holder_name'];
```

Comment:
Two level ternaries (as much as I'd love them) are not that readable, use an `if` block here.